c7n/filters/offhours.py

```diff
@@ -495,9 +497,10 @@ def parse(self, tag_value):
         return schedule
 
     @staticmethod
-    def has_resource_schedule(tag_value):
+    def has_resource_schedule(tag_value, time_type):
         raw_data = ScheduleParser.raw_data(tag_value)
-        return 'on' in raw_data and 'off' in raw_data
+        # note time_type is set to 'on' or 'off' and raw_data is a dict
```

Comment:
fair enough, we typically targeted both off and on to the same value. its a question of adding a new use case of whether thats an either instead of and, and if users are likely to get the and case wrong and slip accidentally into an either scenario. anyways, I'm game for it being either.